<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgDataStreamSelection</class>
 <widget class="QDialog" name="CQDlgDataStreamSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Data Stream Selection</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Data stream type</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="qqTypeCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>391</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>30</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Object / items to record</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="qqItemCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>391</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>30</number>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>130</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqTypeCombo</tabstop>
  <tabstop>qqItemCombo</tabstop>
  <tabstop>qqOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
